StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement+Internal.swift

```diff
@@ -151,6 +151,9 @@ extension EmbeddedPaymentElement: EmbeddedPaymentMethodsViewDelegate {
     func embeddedPaymentMethodsViewDidTapPaymentMethodRow() {
         guard let selectedFormViewController else {
             // If the current selection has no form VC, there's nothing to do
```

Comment:
We should update this comment. Specifically we should mention why there is no selectedFormViewController when in immediateAction mode

Reply:
Ah yeah, updated!